util/content.go

```diff
@@ -213,3 +214,10 @@ func GetContent(contentid string, db *gorm.DB, u *User) (Content, error) {
 	}
 	return content, nil
 }
+
+func CheckContentTypeIsMultipartFormData(header http.Header) error {
```

Comment:
I was hoping you make it an actual HTTP middleware, like the others

Reply:
will that work though? Because /content endpoint also has `handleAddIpfs` which does not need a multipart/form-data. 

Same is the case for `handleAddCar`

Reply:
@anjor this is an example I did for `car` endpoint - https://github.com/application-research/estuary/blob/dev/handlers.go#L161